clients/java/pom.xml

```diff
@@ -95,6 +95,24 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.junit.vintage</groupId>
+      <artifactId>junit-vintage-engine</artifactId>
```

Comment:
💭 I was not able to see the usage of this package and `junit-jupiter-engine` in the test class. Are they really needed to be added as dependency? 🤔